src/main/resources/public/ts/directives/calendar-item/calendar-item.ts

```diff
@@ -2,14 +2,15 @@ import {model, moment, ng} from "entcore";
 import {ROOTS} from "../../core/const/roots";
 import {Calendar} from "../../model";
 import {IScope} from "angular";
+import {Subject} from "rxjs";
 import {ICalendarService} from "../../services";
-import {DateUtils} from "../../utils/date.utils";
+// import {DateUtils} from "../../utils/date.utils";
```

Comment:
for further use ([MC-210](https://jira.support-ent.fr/browse/MC-210))

Reply:
Je comprend pas pourquoi est ce que tu aurais besoin de mettre du code en commentaire pour un future ticket